src/workflow/confirm_locktime.c

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "confirm_locktime.h"
+
+#include "blocking.h"
+#include "status.h"
+
+#include <ui/components/confirm_transaction.h>
+#include <ui/screen_stack.h>
+#include <util.h>
+#include <workflow/confirm.h>
+
+bool workflow_confirm_locktime(uint32_t locktime, bool rbf)
+{
+    char formatted_locktime[100] = {0};
+    if (rbf) {
+        snprintf(
+            formatted_locktime,
+            sizeof formatted_locktime,
+            "Locktime set to block:\n%lu\nTransaction is RBF",
```

Comment:
Did you test this on a device?I think `%lu` might not work.

Reply:
Yes, also seems to work fine for a large number, like a timestamp. But as I said initially, I am very  unsure of the conversion.